### PR TITLE
Feature/include apk link

### PR DESCRIPTION
**Pull request by j8seangel**
Include apk link below the store's links as an example of implementation, please don't merge until design review:

image

Fix: language selector fixed on desktop

### CODE REVIEW

File: src/constants/landing.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { FACEBOOK_WIDGET_API, GOOGLE_PLUS_ONE_WIDGET_API, TWITTER_WIDGET_API } f
 
 export const GFW_ASSETS_PATH = process.env.REACT_APP_GFW_ASSETS_PATH;
 
+export const DOWNLOAD_APK_LINK = "https://rink.hockeyapp.net/api/2/apps/2394979ddd9d415c9b0e466656757d5d/app_versions/9?format=apk&avtoken=510e2758876fcd8bbd275f0eff3ff33bb3e36984&mctoken=18aa5ac7b7a92bf4840c3eb4ef096d054bdff8c8"
```
Comment (sorodrigo) [inline src/constants/landing.js]: Needs to be an Env var so we can update it without changing code.

File: src/styles/components/_c-landing.scss [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
       @media (min-width: 1171px) {
         right: 235px;
+        position: absolute;
+        top: -58px;
```
Comment (sorodrigo) [inline src/styles/components/_c-landing.scss:24]: We need to think about what are we going to do with this. We can’t keep adding tech debt forever, eventually we’ll need to clean up the code.

File: src/constants/landing.js (hunk 1 above)
Comment (j8seangel) [reply src/constants/landing.js]: 👍

File: src/styles/components/_c-landing.scss (hunk 2 above)
Comment (j8seangel) [reply src/styles/components/_c-landing.scss:24]: This selector has been a pain from the beginning, I agree with you but we have other priorities to focus on for now what adds more value.

**Pull request merged**